content/docs/minion-v0.4/tag_cache.md

```diff
@@ -150,7 +151,7 @@ Some extra hardware optimisation techniques are used for better cache capacity a
 #### Bottom-up search order
 
 The tag tree is searched in a bottom-up order.
-If a tag is found in a *tag table* node, there is no need to search higher *tag map* nodes.
+If a tag is found in a *tag table* node, there is no need to search deeper *tag map* nodes.
```

Comment:
I am not sure it is deeper unless you put the root of the tree at the bottom of the figure. However, I put it at the top.